src/adapters/database/impl/NeDBAdapter.ts

```diff
@@ -163,6 +169,25 @@ const hasToken = async (token: string): Promise<boolean> => {
     return !!savedToken && savedToken.token === token;
 };
 
+const setTimezone = (id: string, timezone: string): Promise<void> => {
+    Log.debug("Setting timezone:", timezone);
+    const config = {id, timezone};
+    const collection = getCollection(Setting.TIMEZONE);
+    return promisifyNeDB<void>(collection.update.bind(collection))({id}, config, {upsert: true});
+};
+
+const getTimezone = async (id: string): Promise<string> => {
+    const collection = getCollection(Setting.TIMEZONE);
+    const query = {id};
+    const document = await promisifyNeDB<TimezoneConfig>(collection.findOne.bind(collection))(query);
+    Log.debug("Retrieved timezone", document?.timezone, `for id ${id}`);
```

Comment:
Maybe a similar space here (not sure how it looks though)

`Log.debug("Retrieved timezone ", document?.timezone, `for id ${id}`);`

Reply:
[see other comment!](https://github.com/kobayashley/blues/pull/76#discussion_r632217573)